src/delivery-options/DeliveryOptions.vue

```diff
@@ -30,14 +30,16 @@
 </template>
 
 <script>
+/* eslint-disable no-console */
```

Comment:
Er zit geen console in dit bestand, is dit dan nog nodig?

Reply:
goeie